src/pages/Transfer/CrossChainTransferForm/components/ChainIcon/ChainIcon.tsx

```diff
@@ -0,0 +1,45 @@
+import { forwardRef, ForwardRefExoticComponent, RefAttributes } from 'react';
+
+import { IconProps } from '@/component-library/Icon';
+
+import { StyledFallbackIcon } from './ChainIcon.style';
+import { INTERLAY, KINTSUGI, KUSAMA, POLKADOT } from './icons';
+
+type ChainComponent = ForwardRefExoticComponent<IconProps & RefAttributes<SVGSVGElement>>;
+
+const chainsIcon: Record<string, ChainComponent> = {
+  INTERLAY,
```

Comment:
@danielsimao thoughts on the casing of this? I matched it to the tickers, but arguably title case is better

Reply:
Since you are reusing the icons in CoinIcon, I would make these to those.

Reply:
I'm not sure about that—we are for some chains, not for others (we don't have it yet, but statemine/statemint isn't a token logo). We also can't be certain that we'll always maintain the relationship between chain icons and token icons. I'm not keen on the duplication though. What about if we moved all icons to their own directory, and created three levels:

- Token only (e.g. USDT)
- Chain only (e.g. Statemine)
- Token and chain (e.g. Polkadot/DOT)

That way the use case for every icon would be clear, and without duplication.

Reply:
ah I get it now. Yes that could be an alternative.

Reply:
Yes, it can be a backlog task—it's not urgent. I'd want to make these changes in a separate PR anyway as they'll affect files outside the scope of this task.

I'll add a ticket to the de-fi hub milestone. It's a quick change, and prevents us accumulating tech debt so I'd much prefer to get it done now rather than leave it on the backlog.